addons/test_mail_full/tests/test_mail_performance.py

```diff
@@ -316,3 +316,13 @@ def test_portal_message_format_monorecord(self):
             res = message.portal_message_format(options={'rating_include': True})
 
         self.assertEqual(len(res), 1)
+
+    def test_rating_performance(self):
+        record_rating = self.record_ratings[0]
+        mail_record = self.env['mail.message'].create([{
+            'model': record_rating._name,
+            'res_id': record_rating.id,
+        } for x in range(10)])
+
+        with self.assertQueryCount(default=10):
```

Comment:
Could be great to add the runbot query count before and after the changes in the commit message